interface/service.proto

```diff
@@ -33,16 +33,10 @@ import "externalobject.proto";
 
 /* Catena's API */
 
-message GeneralRequest {
+message EmptyRequest {
```

Comment:
is this what JP recommended?

Reply:
- copilot just told me that this is how to do it, so comment rescinded ;-)

Reply:
JP did recommend this.